datadog-integrations-aws-handler/datadog-integrations-aws.json

```
@@ -79,6 +79,10 @@
     "IntegrationID": {
       "description": "An identification value that represents this integration object. Combines the AccountID, RoleName, and AccessKeyID. This shouldn't be set in a stack.",
       "type": "string"
+    },
+    "ExternalIDSecretName": {
+      "description": "The name of the AWS SecretsManager secret we create in your account to hold this integration's external_id. Defaults to DatadogIntegrationExternalID. Cannot be referenced from created resource.",
```

Wasn't sure about the language here, does this seem reasonable @zippolyte?

---

Sounds good, just a small nit
```suggestion
      "description": "The name of the AWS SecretsManager secret we create in your account to hold this integration's external_id. Defaults to `DatadogIntegrationExternalID`. Cannot be referenced from created resource.",
```